README.md

```diff
@@ -644,9 +670,9 @@ After user authorizes to use their payment information on Android Pay chooser sc
 
 #### Full Wallet [⤴](#table-of-contents)
 
-Masked Wallet information has the shipping address and billing address, to be used for calculating the exact total purchase price. After the app obtains the Masked Wallet, it should present a confirmation page showing the total cost (`Storefront.Checkout#getTotalPrice`) of the items purchased in the transaction, total taxes (`Storefront.Checkout#getTotalTax`) and option to select shipping method if checkout requires shipping (`StoreFront.Checkout#getRequiresShipping`). 
+Masked Wallet information includes the shipping address and billing address, to be used for calculating the exact total purchase price. After the app obtains the Masked Wallet, it should present a confirmation page showing the total cost (`Storefront.Checkout#getTotalPrice`) of the items purchased in the transaction, total taxes (`Storefront.Checkout#getTotalTax`), and an option to select shipping method if the checkout requires shipping (`StoreFront.Checkout#getRequiresShipping`).
 
-When the user confirms the order Full Wallet information should be requested (`PayHelper#requestFullWallet`) to obtain payment token required to complete checkout. To request Full Wallet information you must provide updated cart with the exact total purchase price and authorized Masked Wallet by user:
+When the user confirms the order, Full Wallet information should be requested (`PayHelper#requestFullWallet`) to obtain the payment token required to complete checkout. To request Full Wallet information, you must provide the updated cart with the exact total purchase price and the Masked Wallet that was authorized by the user:
```

Comment:
I added "that was authorized by the user" here to clarify -- but is that accurate?